common/pipelinegen/config_builder.go

```diff
@@ -196,6 +199,37 @@ func applyRootPipelineForSignal(currentConfig *config.Config, signal common.Obse
 	}
 }
 
+func insertServiceGraphPipeline(currentConfig *config.Config) {
+	// Add the service graph exporter to expose the service graph metrics to prometheus
+	currentConfig.Exporters["prometheus/servicegraph"] = config.GenericMap{
+		"endpoint":  "localhost:9090",
```

Comment:
Should we keep the port configurable or atleast declare it as a const (for future configurability) in case end user want it to be configurable?